openvdb_houdini/CMakeLists.txt

```diff
@@ -48,11 +48,6 @@ houdini_get_default_install_dir():
   Windows: $ENV{HOMEDRIVE}$ENV{HOMEPATH}\Documents\houdiniX.X
 If OFF, CMAKE_INSTALL_PREFIX is used.]=] ON)
 
-# SESITAG CMake options
-option(OPENVDB_HOUDINI_ENABLE_SESI_TAG [=[
```

Comment:
Shouldn't we wait until we deprecate support for H17 and H17.5 to actually remove this option?

Reply:
It's not removed, just deprecated (and no longer visible as an option). In my opinion it makes sense to do this now seeing as this does nothing for Houdini 18/future versions

Reply:
Ok, makes sense, thanks.